### ISSUE TITLE
EDIT button on BY URL is not working

### ISSUE THREAD

**Issue by tkrop** [zalando/zally#840]
When providing an API by URL, the EDIT button is not working. The VALIDATE button works fine.

**Comment by vadeg** [comment]
@tkrop is it still the case?

**Comment by bksun** [comment]
@vadeg yes. It gives `cors` error even after using `cors() `module on server
